addons/event_sale/i18n/event_sale.pot

```diff
@@ -203,6 +203,12 @@ msgstr ""
 msgid "Event Tickets"
```

Comment:
Could you update the reproduction steps in the commit and PR message? They are more complicated than they need to be. Ideally you want the least number of steps and modules involved in reproduction as possible.

You can just say:
```
- Create a sale order for a registration
- Set the 'event' field empty for that sale order line
- Confirm the order
-> Traceback as event_id is not set on the event registration 
```